client/src/components/Reviews.jsx

```diff
@@ -56,7 +56,10 @@ class Reviews extends React.Component {
           {review.review_description}
         </div>
         <div className="review-footer">
-          <p className="likes_count">{review.likes_count}</p>
+          <p className="likes_count">
+            {review.likes_count}
+            { ' likes' }
```

Comment:
I honestly have no idea what this does - I presume it does what it needs to, but I've never seen annotation like this in React before, e.g. the `{ ' likes' } `

Reply:
Actually, I think you're just inserting placeholder text on further review - I'd just remove the extra space. May be worth doing an accounting if something has 1 "like" for singular vs plural, but that's pretty nitpicky + not worth more than 5 mins